OJP/OJP_JourneySupport.xsd

```diff
@@ -97,7 +97,11 @@
 					<xs:documentation>Contains the Vehicle reference of the vehicle. In Transmodel this may be the VEHICLE Id.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element ref="OperatorRef" minOccurs="0"/>
+			<xs:element name="Operators" type="OperatorsRefStructure" minOccurs="0">
```

Comment:
Shouldn't this be named `name="OperatorRefs"` then?

Reply:
adapted